ingestors/documents/pdf.py

```diff
@@ -20,8 +28,13 @@ def ingest(self, file_path, entity):
         """Ingestor implementation."""
         try:
             self.parse_and_ingest(file_path, entity, self.manager)
+        except pikepdf._qpdf.PasswordError as pwe:
+            raise ProcessingException(
+                "Could not extract PDF file. The PDF is protected with a password. Try removing the password protection and re-ingesting the documents."
```

Comment:
As this error message will be surfaced to end users, maybe we could use "reupload" instead of "reingest"?